Cargo.lock

```diff
@@ -1,3 +1,18 @@
+[root]
```

Comment:
@snd update your cargo, cause you keep adding `[root]` entry in `Cargo.lock`. Latest cargo doesn't do that ;)

Reply:
done